Editor/Unity/DotNetWizard.cs

```diff
@@ -0,0 +1,64 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+ namespace Innoactive.CreatorEditor
+ {
+     /// <summary>
+     /// Modal Window that helps setting the 'API Compatibility Level' to '.Net 4.x'.
+     /// </summary>
+     internal class DotNetWizard : EditorWindow
```

Comment:
Is this a Wizard? To be honest for me this sounds like a DotNetErrorPrompt or Window